detox/detox.d.ts

```diff
@@ -230,6 +230,11 @@ declare global {
              * Retries count. Zero means a single attempt to run tests.
              */
             retries?: number;
+            /**
+             * Arguments that should not be used during retries.
+             * @default ['shard']
+             */
+            noRetryArgs?: string[];
```

Comment:
Oh IDK. Feels like too much work for a user, doesn't it? Could we instead decide by ourselves which args not to use? And just allow for an opt-out? (does the user need _this_ amount of high-granularity control with this?)

Reply:
Detox is still somewhat agnostic to test runner. We already provide defaults for Jest, and that is `['shard']` at the moment.